src/lib/isAbaRouting.js

```diff
@@ -0,0 +1,20 @@
+import assertString from './util/assertString';
+
+// http://www.brainjar.com/js/validation/
+const isRoutingReg = /^[0-9]{9}$/;
+
+export default function isAbaRouting(str) {
+  assertString(str);
+  str = str.trim();
+
+  if (!isRoutingReg.exec(str)) return false;
+
+  const strArr = str.split('');
```

Comment:
If I am not mistaken I don't think it is necessary to split the string's characters into an array -> 
you can access each character in the string via its index directly as well.
I've also tested it locally here and it works with the tests you provided.